contracts/vaults/MetaStable2TokenAuraVault.sol

```diff
@@ -59,34 +60,16 @@ contract MetaStable2TokenAuraVault is MetaStable2TokenVaultMixin {
         uint256 maturity,
         bytes calldata data
     ) internal override returns (uint256 strategyTokensMinted) {
-        DepositParams memory params = abi.decode(data, (DepositParams));
-        MetaStable2TokenAuraStrategyContext memory context = _strategyContext();
-
-        strategyTokensMinted = context.poolContext._deposit({
-            strategyContext: context.baseStrategy,
-            stakingContext: context.stakingContext,
-            deposit: deposit,
-            params: params
-        });
+        _strategyContext().deposit(deposit, data);
```

Comment:
Aren't we losing the return values here?

Reply:
fixed